src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,71 @@
+package lotto.controller;
+
+import java.util.List;
+import java.util.function.Supplier;
+import lotto.model.domain.BonusBall;
+import lotto.model.domain.WinningBalls;
+import lotto.model.service.LottoSellingMachine;
+import lotto.model.service.Lottos;
+import lotto.model.service.RankResult;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public LottoController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void play() {
+        Lottos lottos = sellLoto();
+        WinningBalls winningBalls = choiceWinningNumber();
+        BonusBall bonusBall = choiceBonusNumber(winningBalls);
+        printResult(lottos, winningBalls, bonusBall);
+    }
+
+    private <T> T attempt(Supplier<T> supplier) {
+        while (true) {
+            try {
+                return supplier.get();
+            } catch (IllegalArgumentException e) {
+                outputView.printException(e.getMessage());
+            }
+        }
+    }
+
+    private Lottos sellLoto() {
```

Comment:
```suggestion
    private Lottos sellLotto() {
```
typo가 있는 것 같아 말씀드려봅니다!